vpro-shared-monitoring/src/main/java/nl/vpro/monitoring/config/MonitoringConfig.java

```diff
@@ -92,6 +95,15 @@ public PrometheusMeterRegistry globalMeterRegistry() {
                 new DiskSpaceMetrics(new File(folder)).bindTo(registry);
             }
         }
+        if (properties.isMeterLocks()) {
+            Gauge.builder("locks.currentCount", ObjectLockerAdmin.JMX_INSTANCE, ObjectLockerAdmin::getCurrentCount)
+                .description("The current number of locked objects")
+                .register(registry);
+            Gauge.builder("locks.maxConcurrency", ObjectLockerAdmin.JMX_INSTANCE, ObjectLockerAdmin::getMaxConcurrency)
```

Comment:
Same here lower case only. And what does max.concurrency mean?

Reply:
I put the meaning in the description.